scrapy/downloadermiddlewares/redirect.py

```diff
@@ -71,6 +71,9 @@ def process_response(self, request, response, spider):
             return response
 
         location = safe_url_string(response.headers['location'])
+        if response.headers['location'].startswith(b'//'):
```

Comment:
Could you make sure `response.headers['location']` is not `None` before this? I don't think any current Scrapy component would set the header to `None`, but it's _possible_ (perhaps by a user-defined middleware) and it would trigger an `AttributeError`.
Also, could you rename `location` to `Location`? Not an issue in practice since lookups are caseless in this case, just to comply with the header name as described in [RFC 7231](https://tools.ietf.org/html/rfc7231#section-7.1.2).
Sorry I missed these the first time.

Reply:
Done! :)